tools/c7n_gcp/tests/test_loadbalancer.py

```diff
@@ -71,6 +71,52 @@ def test_loadbalancer_url_map_get(self):
         self.assertEqual(instance['kind'], 'compute#urlMap')
         self.assertEqual(instance['fingerprint'], 'GMqHBoGzLDY=')
 
+    def test_loadbalancer_url_map_delete(self):
+        project_id = 'custodian-test-project-0'
+        session_factory = self.replay_flight_data('lb-url-map-delete',
+                                                  project_id=project_id)
+        base_policy = {'name': 'lb-url-map-delete',
+                       'resource': 'gcp.loadbalancer-url-map'}
+
+        policy = self.load_policy(
+            dict(base_policy,
+                 filters=[{'type': 'value',
+                           'key': 'name',
+                           'op': 'contains',
+                           'value': 'url-map'}],
+                 actions=[{'type': 'delete'}]),
+            session_factory=session_factory)
+        resources = policy.run()
+        self.assertEqual(1, len(resources))
+        self.assertEqual('url-map-0', resources[0]['name'])
+
+        if self.recording:
+            sleep(3)
+
+        policy = self.load_policy(base_policy, session_factory=session_factory)
```

Comment:
use an api client to verify policy behavior, not an a rerun of the same policy.

Reply:
Updated